gd-glm/src/containers/Layout/Footer.jsx

```diff
@@ -5,7 +5,7 @@ import {
     Grid,
     makeStyles
 } from '@material-ui/core';
-
+import packageJson from '../../../package.json';
```

Comment:
Can you instead import `package.json` version in `webpack.config.common.js` in the repo root and assign it to an environment variable, e.g. `VERSION`, with the `DefinePlugin`? An example of the plugin is in `gd-gltg/webpack.config.dev.js`.

This way, all projects will have access to this variable and can use it in any place that needs its value via `process.env.<variable-name>`.

Reply:
Sure that sounds better. Ill do it that way